flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveLookupFunction.java

```diff
@@ -47,29 +59,46 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Properties;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import static org.apache.flink.connectors.hive.HiveTableFactory.LOOKUP_JOIN_CACHE_TTL;
+import static org.apache.flink.connectors.hive.HiveTableFactory.LOOKUP_JOIN_PARTITION;
+import static org.apache.flink.table.catalog.hive.util.HivePartitionUtils.getPartitionByPartitionSpecs;
+import static org.apache.flink.table.catalog.hive.util.HivePartitionUtils.getTableProps;
+import static org.apache.flink.table.catalog.hive.util.HivePartitionUtils.toHiveTablePartition;
+import static org.apache.flink.table.catalog.hive.util.HivePartitionUtils.validateAndParsePartitionSpecs;
+import static org.apache.flink.util.Preconditions.checkNotNull;
+
 /**
- * Lookup table function for filesystem connector tables.
+ * Lookup table function for Hive connector tables.
  */
-public class FileSystemLookupFunction<T extends InputSplit> extends TableFunction<RowData> {
+public class HiveLookupFunction<T extends InputSplit> extends TableFunction<RowData> {
```

Comment:
Why do this? Why not Filesystem also have this lookup capability?

Reply:
The  `FileSystemLookupFunction ` only used in HiveTableSource.

Reply:
But we can let Filesystem supports this

Reply:
I think we should a better code design, to make the components more decoupled.